static/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "short_name": "The Blue Alliance",
+  "name": "The Blue Alliance",
+  "start_url": "/",
+
+  "gcm_sender_id": "103953800507"
```

Comment:
Ah, you're right. This is the common one. The one above is the one we shouldn't check in